app/views/admin/report_notes/_report_note.html.haml

```diff
@@ -1,18 +1,24 @@
-.report-notes__item
+.report-notes__item{ id: dom_id(report_note) }
   = image_tag report_note.account.avatar.url, class: 'report-notes__item__avatar'
 
   .report-notes__item__header
     %span.username
       = link_to report_note.account.username, admin_account_path(report_note.account_id)
-    %time.relative-formatted{ datetime: report_note.created_at.iso8601 }
-      = l report_note.created_at.to_date
+    %a.timestamp{ href: "##{dom_id(report_note)}" }
+      %time.relative-formatted{ datetime: report_note.created_at.iso8601 }
+        = l report_note.created_at.to_date
 
   .report-notes__item__content
-    = linkify(report_note.content)
+    - if report_note.is_a?(InstanceNote)
+      = report_note.formatted_content
+    - else
+      = linkify(report_note.content)
 
   - if can?(:destroy, report_note)
     .report-notes__item__actions
       - if report_note.is_a?(AccountModerationNote)
         = table_link_to 'delete', t('admin.reports.notes.delete'), admin_account_moderation_note_path(report_note), method: :delete
+      - if report_note.is_a?(InstanceNote)
+        = table_link_to 'delete', t('admin.reports.notes.delete'), admin_instance_note_path(instance_id: report_note.domain, id: report_note.id), method: :delete
```

Comment:
This is a little clunky because I nested the notes under the instances routes, so:

```
/admin/instances/:instance_id/notes/:id
```

Where as both Report Notes and Account Moderation Notes are top-level routes in admin:

```
/admin/account_moderation_notes/:id
/admin/report_notes/:id
```

I think those probably should have been nested originally.